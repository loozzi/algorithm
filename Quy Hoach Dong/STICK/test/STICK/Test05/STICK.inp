10

31
74 48
10 35
97 73
72 10
63 16
73 48
30 79
58 92
17 98
16 64
77 73
89 89
20 66
68 75
38 41
13 44
4 69
87 84
88 48
60 72
34 30
96 61
58 89
41 54
17 18
55 43
5 99
59 61
6 48
41 9
58 25

21
11 14
27 96
75 68
28 48
72 5
21 25
48 42
53 40
98 9
61 50
33 38
60 35
59 54
24 68
48 41
27 45
53 72
52 81
27 25
27 38
32 92

25
28 19
69 23
77 44
50 80
64 13
58 96
22 47
61 19
41 33
25 74
7 26
90 44
96 32
60 85
8 86
66 91
93 14
100 95
66 46
71 32
71 74
22 63
75 20
2 38
15 19

34
89 79
24 9
16 29
40 73
50 77
28 97
64 31
76 94
80 78
62 19
8 64
16 40
51 15
15 47
18 72
75 33
54 1
44 52
56 95
7 53
40 62
11 52
32 3
64 69
31 35
59 76
38 39
60 63
95 27
83 66
43 28
59 11
30 59
17 73

29
82 16
13 7
41 32
25 18
75 49
82 54
24 7
41 71
49 22
5 53
13 48
47 21
39 21
6 86
74 32
29 66
46 98
34 28
92 88
54 90
33 12
77 28
32 27
99 48
25 88
47 41
29 96
25 95
96 54

43
27 47
97 52
57 79
34 55
83 20
21 39
97 33
43 52
43 65
47 19
83 10
15 66
36 1
46 68
68 34
85 52
17 74
54 89
19 35
85 32
78 41
1 74
55 23
52 85
78 82
57 43
65 28
14 31
86 10
39 44
36 32
41 32
30 94
11 45
95 80
71 38
52 71
91 67
82 69
34 62
27 7
43 77
67 13

10
82 38
13 72
80 67
94 39
39 16
62 89
10 47
45 28
54 47
59 90

15
16 72
30 24
6 63
5 22
27 94
35 41
30 28
59 68
87 32
71 74
74 86
13 8
49 19
66 20
43 48

44
23 15
95 27
80 85
44 7
97 79
96 93
47 33
29 34
87 93
58 64
24 86
12 40
69 6
12 64
72 19
8 42
18 98
56 63
33 5
47 11
92 11
47 11
4 16
46 90
81 15
88 4
99 73
6 82
67 32
35 45
50 75
71 23
82 85
66 51
59 44
2 86
74 48
82 50
83 96
47 47
20 6
53 45
8 78
3 78

6
73 22
42 81
28 37
9 34
27 45
33 12
