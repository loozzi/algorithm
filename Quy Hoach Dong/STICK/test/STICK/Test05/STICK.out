8
7
6
8
6
8
4
3
12
3
